CONTRIBUTE.md

```diff
@@ -38,7 +38,7 @@ The official way of running them is through Cucumber-ruby and Aruba. Ruby and Bu
 
     $ git submodule update --init
     $ bundle
-    $ rm -rf doc; ARUBA_REPORT_DIR=doc cucumber features/cucumber-tck -r features
+    $ rm -rf doc; ARUBA_REPORT_DIR=doc cucumber -p compliance
```

Comment:
Specifying the ARUBA_REPORT_DIR is causing this to fail on my machine.
Had to add bcat and rdiscount to the gemfile just to get it to run.
Then every test is hitting
```
undefined method `line' for  #<Cucumber::...>
```
Where it says the error is keeps changes with each test